airbyte-integrations/base-python/airbyte_protocol/entrypoint.py

```diff
@@ -1,28 +1,34 @@
 import argparse
-import sys
-import tempfile
-import os.path
 import importlib
+import os.path
+import tempfile
+
+import sys
+
+from .integration import ConfigContainer, Source
+from .logger import AirbyteLogger
+
 
-from airbyte_protocol import ConfigContainer
-from airbyte_protocol import Source
-from airbyte_protocol import AirbyteLogger
-from airbyte_protocol import AirbyteLogMessage
-from airbyte_protocol import AirbyteMessage
+def environ_with_default(name, default):
```

Comment:
I'd prefer the built in get method on `dict` which can have a default:

```
→ VAR1=hi python3
Python 3.8.5 (default, Jul 21 2020, 10:48:26)
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ.get("VAR1", "hello")
'hi'
>>> os.environ.get("VAR2", "hello")
'hello'
```

Reply:
oh nice!